### PR TITLE
Create Cloud_service.md

### PR DESCRIPTION
**Pull request by fleadsom**
Cloud_service template for new service requests.
@mcleo-d @git-hub-forwork1 to review

### CODE REVIEW

**Comment by fleadsom** [comment]
linking issue #23

**Comment by mcleo-d** [comment]
Thanks @fleadsom - I have ...

* Assigned to @git-hub-forwork1 to review and merge.
* Added to the CSC kanban as In Progress.
* Assigned you as the PR owner.

Congrats! 🚀🚀🚀

James.

File: .github/ISSUE_TEMPLATE/Cloud_service.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+## New Cloud Service
+
+A new cloud service must follow the standard completion template. The 'definition of done' of a cloud service request is defined as follows:
+
+- [ ] Cloud Platform Target
+- [ ] Service Name
+- [ ] Service Approval Accelerator Definition
+- [ ] Control Framework Definition
+- [ ] Test Cases Created
```
Comment (peterrhysthomas) [inline .github/ISSUE_TEMPLATE/Cloud_service.md:15]: Merge the Control Framework and test cases step together to mirror what we have for redshift

Comment (abdullahgarcia) [reply .github/ISSUE_TEMPLATE/Cloud_service.md:15]: +1

Comment (mcleo-d) [reply .github/ISSUE_TEMPLATE/Cloud_service.md:15]: Thanks for the +1 @abdullahgarcia 👍 

@fleadsom - Please let @abdullahgarcia know when the changes have been done so the PR can be merged into master.

Cheers,

James.

**Comment by abdullahgarcia** [comment]
Looks good!

**Pull request merged**